Modifier fixes

Ivanowicz opened the pull request:
Fixes #231

I'm not sure what to do with *attacked from rear* - corrected the penalty to parry only, but Basic Set seems to forbid it pretty explicitly without special advantages, so leaving it in might lead to some confusion.
Haven't touched the MA hit locations yet, other than changing chest -> torso. 
"to hit Eyes [front/side only, imp/pi/tbb only, *hit torso]" seems a bit too long, though I'm not sure what could be shortened.

Ivanowicz commented:
Separated Martial Arts hit locations into its own subdivision and added the missing locations from MA137. Probably not a good long term solution, but I feel like it's better than just leaving them mixed with the Basic Set ones.

Noticed something with "Best possible" equipment modifier - in B345 it's not a static +4, but a bonus equal to TL/2 rounded down (minimum +2) - given how variable that makes it, I feel like removing the bonus and leaving the players to just use custom modifier for it might make more sense.

rinickolous reviewed (commented):
No issues so far. Let me know when this is ready.

Ivanowicz commented:
A few more questions/notes:

Added categories for combat at different levels and striking at weapons.
Exact penalties for bad footing and distracions are (partially) decided by the GM - I'm thinking of either leaving them out, or maybe having a prompt appear when applying those customizable modifiers.
Removed the damage modifiers in attack, since they don't quite fit with the rest.

Started adding missing melee/ranged/defense modifiers from maneuvers and attack options, and now they don't fit in the modifeir bucket window - those categories probably should be reorganized, but I don't have any good ideas on how exactly.

I want to add hit locations for grappling, since they have different penalties (half the usual, round up). Would a toggle somewhere in the bucket/category be practical, or should I keep it simple and just add a separate category for grappling hit locations?

Ranged move and attack has penalty of "-2 or -Bulk, whichever's worse" - is there any way to get weapon bulk from selected token?
